rego/lib/fugue.rego

```diff
@@ -88,6 +88,8 @@ missing(params) = ret {
     "type": params.resource_type,
     "message": object.get(params, "message", "invalid"),
     "provider": object.get(params, "provider", ""),
+    "attribute": null,
+    "filepath": "",
```

Comment:
I could imagine the `filepath` attribute being useful to indicate "a resource is missing in this template". Should I think of that as out-of-scope for this change though?

Reply:
This field is the "resource filepath". It's for the TF case where we want to show the `.tf` path instead of the module path. Setting it to an empty string like this means that it will always use the input filepath.

Reply:
Ah, thanks 👍